Step_Functions/Statistics_Alpha_Resting/Determine_Significance.R

```diff
@@ -126,6 +126,8 @@ Determine_Significance = function(input = NULL, choice = NULL) {
   print("Test Effect of Experimenter Sex and attractiveness for BAS")
   lm_formula =   paste( "EEG_Signal ~ ((Experimenter_Sex * Behav_Attractiveness * Participant_Sex )", 
                         Personality_Formula_BAS, additional_Factor_Formula, ")", Covariate_Formula)
+ # is it correct that we're not including the main effects in the model, only the interaction term? (For experimenter sex, attractiveness, participant sex?)
+ # I don't see trait-BAS included within the interaction term.
```

Comment:
the formula always includes the main effects if an interaction is added
Personality_Formula_BAS is the trait BAS term

update or remove